packages/mui-codemod/src/v5.0.0/top-level-imports.test.js

```diff
@@ -42,6 +42,23 @@ describe('@mui/codemod', () => {
         const expected = read('./top-level-imports.test/expected.js');
         expect(trim(actual)).to.equal(trim(expected), 'The transformed version should be correct');
       });
+
+      it('should not transform color imports from @mui/material/colors', () => {
+        const input = read('./top-level-imports.test/colors-imports-actual.js');
+        const expected = read('./top-level-imports.test/colors-imports-expected.js');
```

Comment:
Hey @AidanLDev, this doesn't seem right? 🤔 

The file used here is `./top-level-imports.test/colors-imports-actual.js` and `./top-level-imports.test/colors-imports-expected.js`

But you modified `./top-level-imports.test/actual.js` and `./top-level-imports.test/expected.js`